src/components/SubRequirement.vue

```diff
@@ -169,7 +170,7 @@ export default Vue.extend({
     };
   },
   computed: {
-    isCompleted(): boolean {
+    isFulfilled(): boolean {
```

Comment:
Out of curiousity - when/how does isFulfilled ever get altered?

Reply:
isFulfilled does not get altered yet. It only returns false in the meantime, but the idea is to set it up for when we need to calculate when a subreq is completed

Reply:
The purpose right now is to make a distinction between isCompleted, data that is passed for the requirement overall and isFulfilled - whether a subrequirement is completed

Reply:
gotchaa